Bottom: d3383a266f1f3f5d143bb11b5d5e7a23a7ba4091
Top:    4db3cb6c3823f4599401a25ea418e4a96d798e51
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-23 00:44:41 +0000

Disable texturing the board's soldermask


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 4750286..0dfd1b1 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1481,6 +1481,7 @@ GhidDrawMask (int side, BoxType * screen)
   gui->graphics->set_color (out->fgGC, PCB->MaskColor);
   ghid_set_alpha_mult (out->fgGC, thin ? 0.35 : 1.0);
 
+#if 0
   if (first_run) {
     glGenTextures (1, &texture);
     glBindTexture (GL_TEXTURE_2D, texture);
@@ -1508,6 +1509,7 @@ GhidDrawMask (int side, BoxType * screen)
   glTexParameterf (GL_TEXTURE_2D, GL_TEXTURE_WRAP_S, GL_REPEAT);
   glTexParameterf (GL_TEXTURE_2D, GL_TEXTURE_WRAP_T, GL_REPEAT);
   glEnable (GL_TEXTURE_2D);
+#endif
 
   if (!PCB->Data->outline_valid) {
 
@@ -1529,10 +1531,12 @@ GhidDrawMask (int side, BoxType * screen)
   poly_FreeContours (&polygon.NoHoles);
   ghid_set_alpha_mult (out->fgGC, 1.0);
   hidgl_flush_triangles (&buffer);
+#if 0
   glDisable (GL_TEXTURE_GEN_S);
   glDisable (GL_TEXTURE_GEN_T);
   glBindTexture (GL_TEXTURE_2D, 0);
   glDisable (GL_TEXTURE_2D);
+#endif
   hidgl_shader_activate (circular_program);
 
   gui->graphics->use_mask (HID_MASK_OFF);
